dnf-behave-tests/features/repoquery/deps.feature

```diff
@@ -0,0 +1,559 @@
+Feature: Tests for the basic repoquery dependencies functionality:
+    --requires, --provides, --conflicts, --obsoletes
+    --whatrequires, --whatprovides, --whatconflicts, --whatobsoletes
+
+Background:
+ Given I use the repository "repoquery-deps"
+
+
+# --requires
+Scenario: repoquery --requires NAME
+ When I execute dnf with args "repoquery --requires middle1"
+ Then the exit code is 0
+  And stdout is
```

Comment:
Please could you add a description into steps. That it handles Last metadata check or downloading repositories? But it is not relevant to repoquery because it is redirected to stderror.

Reply:
Unrelated to this PR, I'll send another one with that. I've got a branch with some more improvements to it, I'll post it after I give it a bit more testing.

Reply:
The PR is here: https://github.com/rpm-software-management/ci-dnf-stack/pull/628